src/reducers/playerStatsReducer.js

```diff
@@ -0,0 +1,17 @@
+export default (state = [], action) => {
+    switch (action.type) {
+        case 'FETCH_PLAYERS':
+            return action.payload;
+        default:
+            return state;
+    }
+};
```

Comment:
Line 1 to Line 8 are already declared in `fetchPlayersReducer.js`. Duplicate code, please remove.